BenniTask.py

```diff
@@ -0,0 +1,58 @@
+# Defining the minimum and maximum withdrawal amounts
+min_withdrawal = 5 
+max_withdrawal = 1000 
+
+# Initializing the Customer balance
+customer_balance = 1000  # Initial balance 
+
+# Optional input your balance
+customer_balance_input = float(input("Enter your Balance: ",)) 
+
+
+# Function to withdraw money
+def withdraw_money(balance): # balance is just a hint what kind of object/variable we aim our function towards 
```

Comment:
love the comments here, but this can be done in your doc strings, which is your function documentation